### DISCUSSION TITLE
querycoord启动报错问题排查

### DISCUSSION THREAD

**Discussion by evenzzu163** [milvus-io/milvus#34344] [Q&A and General discussion]
![9882ee78d978678e010338d1e917eb45](https://github.com/milvus-io/milvus/assets/171309236/2f6eb85c-c05c-454e-ba4d-60a6da0e88af)
辛苦看下报错原因

**Comment by yhmo** [comment]
"Not enough non-faulty bookies available"

A similar issues:
https://github.com/milvus-io/milvus/issues/13553

Possible reason: some of pulsar bookkeeper pods and pulsar broker pods might run into trouble.
